packages/main/src/plugin/telemetry/telemetry.spec.ts

```diff
@@ -34,8 +36,21 @@ const onDidChangeConfigurationMock = vi.fn();
 const configurationRegistryMock = {
   getConfiguration: getConfigurationMock,
   onDidChangeConfiguration: onDidChangeConfigurationMock,
+  registerConfigurations: vi.fn(),
 } as unknown as ConfigurationRegistry;
 
+const getDefaultTelemetryInfo = vi.fn();
```

Comment:
This mock won't be typed. 

I think you can instead use:
```
const defaultConfigurationMock = {
  getContent: vi.fn(),
  getTelemetryInfo: vi.fn(),
} as unknown as DefaultConfiguration;
```

Then use `vi.mocked(defaultConfigurationMock.getTelemetryInfo)` instead of `getDefaultTelemetryInfo` below